homeassistant/components/plex/services.py

```diff
@@ -114,6 +115,58 @@ def get_plex_server(
     )
 
 
+def _handle_standard_payloads(
+    hass: HomeAssistant, plex_url: typing.Any, plex_server: PlexServer | None
+):
+    # Handle standard media_browser payloads
+    if plex_url.name:
+        server_id = plex_url.host
+        plex_server = get_plex_server(hass, plex_server_id=server_id)
+        if len(plex_url.parts) == 2:
+            if plex_url.name == "search":
+                return {}, plex_server
+            return int(plex_url.name), plex_server
+        # For "special" items like radio stations
+        return plex_url.path, plex_server
+    # Handle legacy payloads without server_id in URL host position
+    if plex_url.host == "search":  # noqa: PLR5501
+        return {}, None
+    return int(plex_url.host), None  # type: ignore[arg-type]
+
+
+def _handle_playqueue_media(
+    playqueue_id: "int | typing.Any",
+    supports_playqueues: bool,
+    plex_server: PlexServer | None,
+    content: "dict[str, int] | typing.Any",
+):
+    if not supports_playqueues:
+        raise HomeAssistantError("Plex playqueues are not supported on this device")
+    try:
+        playqueue = plex_server.get_playqueue(playqueue_id)
+    except NotFound as err:
+        raise MediaNotFound(f"PlayQueue '{playqueue_id}' could not be found") from err
+    return PlexMediaSearchResult(playqueue, content)
+
+
+def _handle_content_as_media_or_playqueue(
```

Comment:
Two things, why typing.Any? and also, do not need quotes around the typehint in python since 3.5 according to phind.com, and everywhere else it is not used, so should remove them.

Reply:
No type Any womp womp

Reply:
Still need to fix